pkg/middleware/auth_proxy/auth_proxy.go

```diff
@@ -182,7 +183,7 @@ func (auth *AuthProxy) Login(logger log.Logger, ignoreCache bool) (int64, *Error
 	if isLDAPEnabled() {
 		id, e := auth.LoginViaLDAP()
 		if e != nil {
-			if e == ldap.ErrInvalidCredentials {
+			if errors.Is(e, ldap.ErrInvalidCredentials) {
```

Comment:
Can we rename `e` -> `err`?

Reply:
@sakjur Renaming it to `err` does work, since it has the `error` interface. I didn't notice first that it does, but yeah.